readthedocs/proxito/views/serve.py

```diff
@@ -98,6 +99,18 @@ def get(self,
                 'Invalid URL for project with versions. url=%s, project=%s',
                 filename, final_project.slug
             )
+            if final_project.has_feature(Feature.PROXITO_SUBPROJECT_PATH):
+                new_subproject_slug, rest = filename.split('/', 1)
+                sub = ProjectRelationship.objects.filter(
```

Comment:
nit: I prefer if we don't chunk names in variables. Here `sub` and in the tests `feat`.